Tasks/AzureTestPlanV0/automatedTestInvoker.ts

```diff
@@ -16,21 +19,32 @@ export async function testInvoker(testsToBeExecuted: string[]) {
 
         switch (testLanguage) {
             case 'Java-Maven':
-                await executemaventests(testsToBeExecuted);
+                exitCode = await executemaventests(testsToBeExecuted);
+                tl.debug(`Execution Status Code for Maven: ${exitCode}`);
                 break;
 
             case 'Java-Gradle':
-                await executegradletests(testsToBeExecuted);
+                exitCode = await executegradletests(testsToBeExecuted);
+                tl.debug(`Execution Status Code for Gradle: ${exitCode}`);
                 break;
 
             case 'Python':
-                await executepythontests(testsToBeExecuted);
+                exitCode =  await executepythontests(testsToBeExecuted);
+                tl.debug(`Execution Status Code for Python: ${exitCode}`);
                 break;
 
             default:
                 console.log('Invalid test Language Input selected.');
         }
 
+        if(exitStatusCode == null){
```

Comment:
we are initializing exitStatusCode to 0, will this condition be valid in that case?

Reply:
Updated